src/modules/shared/components/Footer/Footer.types.ts

```diff
@@ -0,0 +1,28 @@
+import { footerType } from './Footer.constants';
+
+export interface IFooterProps {
```

Comment:
interfaces hoef je niet te prefixen met een `I` mag je gerust instellen in de linter via:
https://typescript-eslint.io/rules/naming-convention/#enforce-that-interface-names-do-not-begin-with-an-i